ui/tests/pages/allocations/task/detail.js

```diff
@@ -60,6 +60,8 @@ export default create({
     clientSource: text('[data-test-volume-client-source]'),
   }),
 
+  hasMeta: isPresent('[data-test-meta]'),
```

Comment:
These are "test pages": a way to have nice syntactic sugar in our actual tests (like `assert.ok(task.hasMeta)`) instead of making the tests themselves feel verbose.